practices/images-best-practices.md

```diff
@@ -0,0 +1,48 @@
+# Best practices when dealing with images for the Web
+
+In a front-end workflow we typically deal with two kinds of images:
+
+* **Icons, logos, buttons and other graphical elements** that are part of the User Interface of the page. These are usually managed by the frontend, and so they are stored with the rest of resources (templates, CSS, JS). Like those resources, they may be shared between many different pages.
+
+* **Content images**. These are unique to a page (or a handful of pages). They will be usually provided from a database or a content management system.
+
+Most best practices are common to both categories.
+
+## Formats
+
+### What format to use
+
+* Use SVGs whenever you want to display a vector image.
+* Use JPEGs for photo images, or images with many little details.
+* Use PNGs for graphs and other images with large areas of the same colour. Also for images that require transparency.
+* Don't use GIFs.
+
+### About WebP
+
+We don't encourage the use of the [WebP format](https://developers.google.com/speed/webp/) as support for it is still not widespread. [MS is not planning to add support to Edge](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/webpimageformat/) at the moment. Firefox is currently working in adding support for it, so we may reconsider after that.
```

Comment:
But:

1. The main title of this document is "Best practices when dealing with images for the Web". Consider that the playbook is also a community resource so it should be applicable for community members who might be reading this.  
2. @dotcode explained the playbook is "What we aspire to do".  

I don't know if we're using WebP anywhere, I haven't seen it in use, but my point is, shouldn't we be recommending formats which deliver the best user experience?